laravel-scout-extended/README.md

```diff
@@ -0,0 +1,56 @@
+# About Algolia's Scout-Extended
+
+Algolia Scout-extended was created bt and mainteained by [Algolia](https://github.com/algolia). The Algolia Search API Client for Laravel. 
+
+## Get started locally
```

Comment:
Have you tested this?

Reply:
How can i test this repo started from a git clone ?

Reply:
let me guess : http://bfy.tw/Mnm7

Reply:
this should be better @nunomaduro i've tested it now and it work